src/DummyConsoleApp/RunProcessAsTask.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>DummyConsoleApp</name>
+    </assembly>
+    <members>
+    </members>
+</doc>

```

Comment:
Going through the whole PR, I suspect all of the documentation is unwanted; the generated files aren't useful, so unless there's some plan here I'd say: delete it all.